charts/stable/common/templates/classes/_pvc.tpl

```diff
@@ -11,7 +11,9 @@ within the common library.
 {{ end -}}
 {{- $pvcName := include "common.names.fullname" . -}}
 {{- if hasKey $values "nameSuffix" -}}
-  {{- $pvcName = printf "%v-%v" $pvcName $values.nameSuffix -}}
+  {{- if not (eq $values.nameSuffix "-") -}}
```

Comment:
is this a common pattern would allowing it to be for Go-like make more sense? e.g. `{{- if not (eq $values.nameSuffix "nil") -}}`

Reply:
Gonna see if it can work to specify it but set it to `""` instead of `"-"`, that would be better I think

Reply:
Nope, the underlying sprig library doesn't differentiate between an empty string and nil (See also https://github.com/Masterminds/sprig/issues/53). Guess we're stuck with `-` for now.